Code/Editor/Plugins/ComponentEntityEditorPlugin/SandboxIntegration.cpp

```diff
@@ -1116,18 +1116,30 @@ bool SandboxIntegrationManager::CanGoToSelectedEntitiesInViewports()
 
 bool SandboxIntegrationManager::CanGoToEntityOrChildren(const AZ::EntityId& entityId) const
 {
+    AZ::Entity* entity = nullptr;
+    AZ::ComponentApplicationBus::BroadcastResult(entity, &AZ::ComponentApplicationRequests::FindEntity, entityId);
+    if (entity == nullptr)
+    {
+        return false;
+    }
+
+    // Skip if this is a UI component; UI components use LyShine's "UI Canvas" and are not visible in the Editor viewport
+    constexpr AZ::Uuid uiElementComponentUuid{ "{4A97D63E-CE7A-45B6-AAE4-102DB4334688}" };
+    const AZ::Component* uiElementComponent = entity->FindComponent(uiElementComponentUuid);
+    if (uiElementComponent != nullptr)
+    {
+        return false;
+    }
```

Comment:
Is there any other way to detect and approach this problem that doesn't introduce a hidden dependency on LyShine code here by hard-coding the UI Element UUID type? For example, based on the description of the PR, could/should this look for missing Transforms as opposed to the presence of UI elements?

Reply:
I like it! Fixed!